### DISCUSSION TITLE
M3/M4 Problem Engraving Text

### DISCUSSION THREAD

**Discussion by Stevyjb** [arkypita/LaserGRBL#1333] [Q&A]
Using M3 The Text is over-burning only on the curved parts of letters and using M4 Dynamic Power the text is under-burning only on the curved part of letters? Can the power be adjusted in settings ?

**Comment by StuartB4** [comment]
You adjust the power setting with the S-MIN and S-MAX settings in Target Image Settings. 

Start LaserGRBL and click Connect.
Have a look in Grbl configuration settings on the Grbl menu and see what your $30 is set to.
If it's 255 thats fine, if it's 1000 that is also fine. That is the maximum power. 

Then when you see S-MAX in Target image settings it should be set to either 255 or 1000 to match. 

Then you can reduce the number in S-MAX for individual jobs to a lower number which lowers the power.

**Reply by Stevyjb** [reply]
![IMG_1626 1](https://user-images.githubusercontent.com/78878950/107820114-2d619000-6d72-11eb-91cf-d22e2b93fe4e.jpg)
I have attached a pic of the problem . Straight lines are fine but on a curve the laser Pulses and Over- burns when using M3 and Under-burns when using M4. The letters are generated from an SGV file
